app/src/main/java/com/example/drawingApp/MainViewModel.kt

```diff
@@ -10,4 +11,13 @@ class MainViewModel : ViewModel() {
     @ColorInt
     var primaryColor = Color.BLACK
 
+    val layerNames: MutableList<String> = mutableListOf()
+    var layerCounter = 1
+
+    var activeBitmap: Bitmap? = null
+
+    var dialogOpen = false
+    var dialogText = ""
```

Comment:
More descriptive. `usersCurrentDialogInput`?

Reply:
renamed to currentDialogText